src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -3947,7 +3947,8 @@ public class Derived : Base
     }
 
     [Fact, CompilerTrait(CompilerFeature.NullableReferenceTypes)]
-    public void RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_09()
+    [WorkItem(61718, "https://github.com/dotnet/roslyn/issues/61718")]
+    public void RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_09A()
```

Comment:
Consider adding a test where the base property is required but is not an auto-property. It seems like we would actually expect that to produce a nullable initialization warning even though we don't know whether it actually corresponds to some non-nullable field in the base.

Also would be good to have a test where the base comes from metadata.

Also would be good to have a test where the base has a non-nullable private field. We can demonstrate that adding `[SetsRequiredMembers]` in a derived type doesn't introduce a warning about the base private field.